### PR TITLE
Fix #1583: Newspaper text line height is wrong

### LINKED ISSUE #1583: Newspaper text line height is wrong
**Version information:**
 - Platform: Windows
 - Version: 22.06.1
 - Build/commit: https://github.com/OpenLoco/OpenLoco/commit/adb6f6ce83cbddbe70d89de3ff34db0c0feef1bf

**Describe the bug**
> first time back in here in a while, figured I'd give the latest master a spin. I haven't seen this mentioned anywhere so it's probably peculiar to my odd Porting Kit windows-on-mac emulation, but I noticed the line-spacing on the newspaper text is squashed

— MK Inst on Discord

Screen_Shot_2022-07-22_at_18 04 56
Screen_Shot_2022-07-22_at_18 05 05

### PR DESCRIPTION
**Pull request by AaronVanGeffen**
The new `wrapString` function processes formatting codes, but does not update the `_currentFontSpriteBase` global at the end of it. This would lead `drawStringLeftWrapped` and `drawStringCentredWrapped` to use the wrong line height. Another bug that was hiding, was an incorrect initial drawing position in both these functions.

v22.06.1:
v22 06 1

This PR:
this_branch

### CODE REVIEW

**Comment by AaronVanGeffen** [comment]
Still not quite right:

This branch:
Screenshot (1)

v22.05:
Screenshot

File: src/OpenLoco/Graphics/Gfx.cpp [hunk 1]
```diff
@@ -1446,6 +1450,7 @@ namespace OpenLoco::Gfx
             }
         }
 
+        _currentFontSpriteBase = font;
```
Comment (duncanspumpkin) [inline src/OpenLoco/Graphics/Gfx.cpp:1453]: Add a comment that this is returning only the final lines font

Comment (AaronVanGeffen) [reply src/OpenLoco/Graphics/Gfx.cpp:1453]: Like this?
```suggestion
        // Note that this is always the font used in the last line.
        // TODO: refactor to pair up with each line, and to not use a global.
        _currentFontSpriteBase = font;
```

Comment (duncanspumpkin) [reply src/OpenLoco/Graphics/Gfx.cpp:1453]: Yup

**Review by duncanspumpkin — APPROVED** [review]

**Pull request merged**